ci/build_container/build_recipes/benchmark.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -x
```

Comment:
Remove `-x` (once done debug)  and add `-e`.

Reply:
done